src/azure-cli/azure/cli/command_modules/policyinsights/tests/latest/test_policyinsights_scenario.py

```diff
@@ -544,326 +576,381 @@ def test_policy_insights_metadata(self):
         assert metadata_resource['name'] == resource_metadata_name
 
     # Test policy attestation CRUD requests at various scopes
-    @record_only()
+    @live_only()
```

Comment:
May I ask why mark this test from ` @record_only()` to `@live_only()`

Reply:
This test stopped working off the recording. Since it had to be re-recorded and it didn't set up its own resources, I had to add a ton of setup to the test. In the course of doing this work, I uncovered a bug in the backend in which attestations don't get deleted, even though the response indicates success. The backend fix is delayed, and it doesn't affect CLI command code (just the test), so I commented out parts of the test that were broken by this backend behavior and set it to live_only(). This test needs to be fixed after the backend is fixed, but the CLI commands should not be blocked on that.

Reply:
Got it, thanks